### DISCUSSION TITLE
GPT-3 chat experience

### DISCUSSION THREAD

**Discussion by YinXiangz** [Zero6992/chatGPT-discord-bot#75] [Q&A] [answered]
I am sorry, I feel that GPT-3 is completely unable to have a good communication with me. The difference between it and the current version of ChatGPT is too large and cannot be trained, so I am willing to replace the cookie every few hours to maintain a good quality of conversation for the current version. Therefore, I do not know if it is possible to provide an update for the current version or how many unknown days are needed for the official adjustment of Cloudflare..

**Comment by Zero6992** [comment, marked as answer]
I'm sorry you had a bad experience, but the session token method no longer works because of Cloudflare. 

I will update as soon as a ChatGPT API becomes available, but please be aware that it may be very unstable because they are watching our every move
